app/kudos/test_models.py

```diff
@@ -1,22 +0,0 @@
-# -*- coding: utf-8 -*-
-"""Test the Kudos models.
```

Comment:
@chibie Good catch that this test isn't implemented. Should we make a ticket to implement this test?